modules/cnf-du-creating-a-machine-config-manifest.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-creating-a-machine-config-manifest_{context}"]
+
+= Creating a machine config manifest
+
+To enable workload partitioning, you need to provide a `MachineConfig` manifest during installation to configure CRI-O and kubelet to know about the workload types.
+
+The manifest, without the encoded file content, looks like this:
+
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 02-master-workload-partitioning
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<01-workload-partitioning-content>
+        mode: 420
+        overwrite: true
+        path: /etc/crio/crio.conf.d/01-workload-partitioning
+        user:
+          name: root
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<openshift-workload-pinning content>
+        mode: 420
+        overwrite: true
+        path: /etc/kubernetes/openshift-workload-pinning
+        user:
+          name: root
+----
+
+The contents of `/etc/crio/crio.conf.d/01-workload-partitioning` should look like this (the `cpuset` value will vary based on the deployment):
```

Comment:
No one will insist that you replace it, but "deployment" in this sentence can be replaced with "hardware" or "hardware specification".